### PR TITLE
Fixes part of #2310 & #2321: Changes in Fraction Interaction and Ratio Interaction Item[a11y]

### LINKED ISSUE #2310: Changes in the Fraction Interaction Item [A11y]
**Describe the bug**

FractionInteraction Item is not fully compatible with some users from the Accessibility point of view.

Issue 1 - The contrast ratio of the fraction interaction view does not match the standard ones, both in the active and inactive states. (assigned to @ARJUPTA)

Issue 2 - There is no content label present for the view when it is first clicked to enter a value (Obsolete Issue)

**To Reproduce**
Steps to reproduce the behaviour:
1. Start the Accessibility Scanner App on your device
2. Open the Oppia Android App and start First Test Topic Exploration
3. Scan the screenshots with the Accessibility Scanner
4. You will see the tests that are not passed by that screen
5. Repeat this for other screens

**Expected behaviour**
All the tests must be passed by scanning the screenshots for both Tablet and Mobile Devices in every orientation.

**Screenshots**

| Issue 1  | Issue 2 | 
|:-------:|:-------:|
| Screenshot_2020-12-17-11-05-59-83_80a9469f53bfe75dbc6e8989be28e2f0 (1) | Screenshot_2020-12-17-11-06-22-04_80a9469f53bfe75dbc6e8989be28e2f0 |
| Screenshot_2020-12-17-11-06-34-67_80a9469f53bfe75dbc6e8989be28e2f0 |
 |

**Device**
- Real Device - RealMe3
- SDK version - 29

**Additional context**
The Screenshots are descriptive enough for understanding the issues but still, if there is a need of clarification then you can comment here. Multiple Pull Requests can also be made for solving these issues.

Certain Views does not have the correct contrast ratios (Inactive state) as mentioned in the screenshots. To solve this, one must find a way to make the views disabled instead of being in non-clickable state only

Reference - [https://github.com/oppia/oppia-android/wiki/Working-on-UI#accessibility-scanner](https://github.com/oppia/oppia-android/wiki/Working-on-UI#accessibility-scanner)

### PR DESCRIPTION
**Pull request by Arjupta**
## Explanation

Fixes part of #2310 & #2321,
( Issue 1 ) Changed Unselected Border color as per Chantel's suggestion, by using making separate background border drawable for fraction interaction
|Before|After|Tests|
|:-:|:-:|:-:|
|Screenshot_1612511519 |Screenshot_1612511167|Screenshot_1612511179 | 
|Screenshot_1612511568 | Screenshot_1612511334 |Screenshot_1612511331|

## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

**Review by rt4914 — CHANGES_REQUESTED** [review]
@ARJUPTA Please add before and after screenshot along with the scanner result screenshot (even if it is successful).

**Comment by Arjupta** [comment]
Blocked on https://github.com/oppia/oppia-android/pull/2548

**Comment by Luffy18346** [comment]
> Blocked on #2548

@ARJUPTA #2548 is merged. So, can you please update this PR?

**Comment by Arjupta** [comment]
> > Blocked on #2548
> 
> @ARJUPTA #2548 is merged. So, can you please update this PR?

Yes, will be updating it soon.

**Comment by Luffy18346** [comment]
> > > Blocked on #2548
> > 
> > 
> > @ARJUPTA #2548 is merged. So, can you please update this PR?
> 
> Yes, will be updating it soon.

ok, thanks

File: app/src/main/res/drawable/add_profile_edit_text_background.xml [hunk 1]
```diff
@@ -2,5 +2,5 @@
 <selector xmlns:android="http://schemas.android.com/apk/res/android">
   <item android:drawable="@drawable/edit_text_black_border" android:state_focused="true" />
   <item android:drawable="@drawable/edit_text_black_border" android:state_pressed="true" />
-  <item android:drawable="@drawable/edit_text_background_border" />
+  <item android:drawable="@drawable/add_profile_edit_text_background_border" />
 </selector>
```
Comment (rt4914) [inline app/src/main/res/drawable/add_profile_edit_text_background.xml]: This file is not getting used any where? Can you please confirm and delete this file.

File: app/src/main/res/drawable/add_profile_edit_text_background_border.xml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<shape xmlns:android="http://schemas.android.com/apk/res/android"
+  android:shape="rectangle">
+  <corners android:radius="4dp" />
+  <solid android:color="@color/white" />
+  <stroke
+    android:width="1dp"
+    android:color="@color/grey" />
+</shape>
```
Comment (rt4914) [inline app/src/main/res/drawable/add_profile_edit_text_background_border.xml]: This file has been added but not getting used anywhere. Delete this.

File: app/src/main/res/drawable/edit_text_background_border.xml [hunk 3]
```diff
@@ -5,5 +5,5 @@
   <solid android:color="@color/white" />
   <stroke
     android:width="1dp"
-    android:color="@color/grey" />
+    android:color="#666666" />
```
Comment (rt4914) [inline app/src/main/res/drawable/edit_text_background_border.xml]: Introduce this in colors file
`<color name="oppiaGreyBorderDark">#666666</color>`

**Review by rt4914 — CHANGES_REQUESTED** [review]
@ARJUPTA

File: app/src/main/res/drawable/add_profile_edit_text_background.xml (hunk 1 above)
Comment (Arjupta) [reply app/src/main/res/drawable/add_profile_edit_text_background.xml]: This file was earlier being used by previous input layout I guess. removing it

File: app/src/main/res/drawable/add_profile_edit_text_background_border.xml (hunk 2 above)
Comment (Arjupta) [reply app/src/main/res/drawable/add_profile_edit_text_background_border.xml]: okay

File: app/src/main/res/drawable/edit_text_background_border.xml (hunk 3 above)
Comment (Arjupta) [reply app/src/main/res/drawable/edit_text_background_border.xml]: okay

**Review by rt4914 — APPROVED** [review]
LGTM, thanks.

**Pull request merged**